src/components/Error.js

```diff
@@ -10,29 +11,59 @@ import { HELPLINE_LINK, HELPLINE_TEXT } from '../utils/constants';
 
 import { Button } from './Input';
 
-const Error = () => (
-  <div css={tw`mx-auto p-6 mb-6 text-center`}>
-    <Helmet>
-      <title>Error</title>
-    </Helmet>
-    <h2 css={tw`text-3xl font-bold mb-10`}>Something's wrong on our side.</h2>
-    <div css={tw`max-w-xl mx-auto`}>
-      <p css={tw`text-xl mb-6`}>
-        We're having technical issues right now. The problem should be fixed
-        soon, and in the meantime SAMHSA's national helpline is available 24/7
-        and can assist you with treatment referrals and information.
-      </p>
-      <Button
-        eventLabel="Helpline link from error page"
-        to={`tel:${HELPLINE_LINK}`}
-        as={OutboundLink}
-        primary
-      >
-        <FontAwesomeIcon icon={faPhone} css={tw`mr-2`} className="fa-lg" />
-        {HELPLINE_TEXT}
-      </Button>
-    </div>
-  </div>
-);
+class Error extends Component {
+  headerRef = React.createRef();
+
+  componentDidMount() {
+    this.headerRef.current.focus();
+  }
+
+  render() {
+    const { title, headerText, description, helpLabel } = this.props;
+
+    return (
+      <div css={tw`mx-auto p-6 mb-6 text-center`}>
+        <Helmet>
+          <title>{title}</title>
+        </Helmet>
+        <h1
+          css={tw`text-3xl font-bold mb-10`}
+          ref={this.headerRef}
+          tabIndex="-1"
+        >
+          {headerText}
+        </h1>
+        <div css={tw`max-w-xl mx-auto`}>
+          <p css={tw`text-xl mb-6`}>{description}</p>
+          <Button
+            eventLabel={helpLabel}
```

Comment:
If you want to shave off one prop, we could do:
```suggestion
            eventLabel={`Helpline link from ${title} page`}
```

Reply:
This would change `Helpline link from 404` to `Helpline link from Not found` for the `NoMatch` component, is that ok? I can also fix the capitalization...

Reply:
Yeah I think that's ok - these are just the labels that are associated with custom events we've defined in google analytics. Since we haven't launched yet I think it's ok to change them. Post launch we may want to be more systematic about updating them.